c++/src/kj/exception.c++

```diff
@@ -609,13 +611,16 @@ namespace {
                          "\nstack: ", stringifyStackTraceAddresses(trace),
                          stringifyStackTrace(trace), '\n');
 
+#ifndef KJ_NO_POSIX
   FdOutputStream(STDERR_FILENO).write(message.begin(), message.size());
   _exit(1);
+#endif
 }
 
 }  // namespace
 
 void printStackTraceOnCrash() {
+#ifndef KJ_NO_POSIX
```

Comment:
Instead of removing the body of `printStackTraceOnCrash()`, maybe the whole function should not be defined? It's weird to let people call it but have it do nothing.